lib/AS_BH1750/AS_BH1750A.h

```diff
@@ -77,7 +77,7 @@
 // Device is automatically set to Power Down after measurement.
 #define BH1750_ONE_TIME_LOW_RES_MODE  0x23
 
-#define MAX_U_LONG 4294967295;
+#define MAX_U_LONG 4294967295
```

Comment:
This is not used in that file, should we use it at all?
It is only used in the .cpp, so why not define it there?

Reply:
Being a 3rd party external library I decided to only fix the warnings for now.
It can stay in the h file as it does not collide with any other.